src/mono/wasm/build/WasmApp.Native.targets

```diff
@@ -264,6 +264,8 @@
       <EmscriptenEnvVars Include="PYTHONPATH=$(EmscriptenPythonToolsPath)" Condition="'$(OS)' == 'Windows_NT'" />
       <EmscriptenEnvVars Include="PYTHONHOME=" Condition="'$(OS)' == 'Windows_NT'" />
       <EmscriptenEnvVars Include="EM_CACHE=$(WasmCachePath)" Condition="'$(WasmCachePath)' != ''" />
+      <EmscriptenEnvVars Include="MonoWasmLegacyJsInterop=$(MonoWasmLegacyJsInterop)"/>
```

Comment:
This would be user facing, right? Maybe then `WasmUseLegacyJsInterop`

Reply:
Or `WasmEnableLegacyJsInterop` similar to `WasmEnableThreads`.
Or `FeatureLegacyJsInterop` similar to `FeatureWasmThreads`

I'm slightly confused by `FeatureWasmThreads` and `MonoWasmThreads` should we have just one of them ? 

cc @lambdageek

Reply:
> I'm slightly confused by `FeatureWasmThreads` and `MonoWasmThreads` should we have just one of them ?

Yea we need to clean that up.  I think I initially added `MonoWasmThreads` when prototyping the runtime, and the `FeatureXYZ` stuff got added when the team worked on the runtime packs and SDK.  We don't need both, it's just making things confusing now.

Reply:
I will use `FeatureLegacyJsInterop` and `FEATURE_LEGACY_JS_INTEROP` in <DefineConstants>

Reply:
Changed my mind to `WasmEnableLegacyJsInterop`